packages/ssr/src/rendering/head.ts

```diff
@@ -62,19 +62,19 @@ export const getHeadItems = ({
   })
 
   const preloadFonts: [HeadItemType, string][] = []
-  if ('breakpoints' in theme === false) {
-    // We include all fonts even though it's not necessary.
-    // While this is not the ideal long-term solution, it does have a few benefits:
-    // - It's easier to cache the font stylesheet across pages
-    // - It simplifies style variable setup in theme.ts
-    // - It ensures the same behaviour as in our editor
-    // - It increases the chance that there's a font to be used by our
-    //   reset stylesheet (font-family: var(--font-sans))
-    // For most apps, the overhead of including all fonts is negligible and
-    // it doesn't add a lot of bytes to our stylesheet.
+  Object.values(themes).forEach((theme) => {
```

Comment:
Ideally, we should make sure not to add the same font twice if it's defined in multiple themes. I guess it's an existing issue though since it could exist multiple times in the same theme as well 🤷‍♂️